src/templates/layouts/event/index.tsx

```diff
@@ -27,16 +28,40 @@ export const Event = ({
   link,
   additionalInfo,
   eventCTA,
+  emailCTA,
+  howToSignUp,
   body,
   listing,
 }: FormattedEvent) => {
   const [showRecurringEvents, setShowRecurringEvents] = useState(false)
+  const [mostRecentDate, setMostRecentDate] = useState(null)
+  const [showAllEvents, setShowAllEvents] = useState(false)
+  const initialFormattedDates = formatDateObject(datetimeRange).slice(0, 5)
+  const [currentFormattedDates, setCurrentFormattedDates] = useState(
+    initialFormattedDates
+  )
+  const formattedDates = formatDateObject(datetimeRange)
+
+  useEffect(() => {
+    // Calculate the most recent date when the component mounts
+    const allDates = formatDateObject(datetimeRange)
```

Comment:
is this var necessary with `formattedDates` above?

Reply:
Modified to just use formatted dates